content/nginxaas-azure/quickstart/disaster-recovery.md

```diff
@@ -0,0 +1,207 @@
+- Terraform 1.3+ and AzureRM provider 4.23+.
+
+> **Note**: Each NGINX deployment **must run on separate subnets and non-overlapping address spaces**. This is critical for [Virtual Network (VNet) peering](https://learn.microsoft.com/en-us/azure/virtual-network/how-to-configure-subnet-peering) between the two regions. For example:
+>
+>  - Region 1 (Virtual Network - 1 Address Space): `10.0.0.0/16`
+>  - Region 2 (Virtual Network - 2 Address Space): `10.1.0.0/16`
+
+---
+
+
+## Step 1: Deploy Prerequisite Infrastructure
+
+Each region requires its own resource group, VNet, subnet, public IP, network security group, and user-assigned identity. Example allocation in the prerequisites module:
+
+```hcl
+# Region 1
+resource "azurerm_virtual_network" "deployment_primary_vnet" {
+  address_space = ["10.0.0.0/16"]
+  # other config...
+}
+resource "azurerm_subnet" "deployment_primary_subnet" {
+  address_prefixes = [cidrsubnet("10.0.0.0/16", 8, 0)] # results in 10.0.0.0/24
+}
+
+# Region 2
+resource "azurerm_virtual_network" "deployment_secondary_vnet" {
+  address_space = ["10.1.0.0/16"]
+}
+resource "azurerm_subnet" "deployment_secondary_subnet" {
+  address_prefixes = [cidrsubnet("10.1.0.0/16", 8, 0)] # results in 10.1.0.0/24
+}
+```
+---
+
+## Step 2: Deploy NGINXaaS for Azure Deployment in Each Region
+
+```hcl
+resource "azurerm_nginx_deployment" "deployment_primary_nginxaas" {
+  name                = var.name_primary
+  resource_group_name = var.resource_group_primary
+  location            = var.location_primary
+  ...
+  network_interface {
+    subnet_id = azurerm_subnet.deployment_primary_subnet.id
+  }
+}
+resource "azurerm_nginx_deployment" "deployment_secondary_nginxaas" {
+  name                = var.name_secondary
+  resource_group_name = var.resource_group_secondary
+  location            = var.location_secondary
+  ...
+  network_interface {
+    subnet_id = azurerm_subnet.deployment_secondary_subnet.id
+  }
+}
+```
+
+
+---
+
+## Step 3: Peer the VNets
+
+```hcl
+resource "azurerm_virtual_network_peering" "vnet_primary_to_vnet_secondary" {
+  name                      = "peering-vnet-primary-to-vnet-secondary"
+  resource_group_name       = var.resource_group_primary
+  virtual_network_name      = azurerm_virtual_network.deployment_primary_vnet.name
+  remote_virtual_network_id = azurerm_virtual_network.deployment_secondary_vnet.id
+}
+resource "azurerm_virtual_network_peering" "vnet_secondary_to_vnet_primary" {
+  name                      = "peering-vnet-secondary-to-vnet-primary"
+  resource_group_name       = var.resource_group_secondary
+  virtual_network_name      = azurerm_virtual_network.deployment_secondary_vnet.name
+  remote_virtual_network_id = azurerm_virtual_network.deployment_primary_vnet.id
+}
+```
+
+
+---
+
+## Step 4: Configure Upstreams
+
+Deploy upstream VMs in a subnet separate from the NGINXaaS deployment subnet in the primary region. Example:
```

Comment:
Since it's important to the setup, maybe highlight the primary region?
```suggestion
Deploy upstream VMs in a subnet separate from the NGINXaaS deployment subnet in the **primary region**. Example:
```